packages/store/ts/storeEvents.ts

```diff
@@ -0,0 +1,6 @@
+export const storeEvents = [
+  "event StoreDeleteRecord(bytes32 table, bytes32[] key)",
+  "event StoreSetField(bytes32 table, bytes32[] key, uint8 schemaIndex, bytes data)",
+  "event StoreSetRecord(bytes32 table, bytes32[] key, bytes data)",
+  "event StoreEphemeralRecord(bytes32 table, bytes32[] key, bytes data)",
+] as const;
```

Comment:
this is the main thing I wanted to import from another package into the browser bundle